enforcer-parent/commons/src/main/java/org/wso2/choreo/connect/enforcer/commons/model/ParameterResolver.java

```diff
@@ -29,14 +29,16 @@
 
 class ParameterResolver {
 
-    private static final Pattern PARAMETER_PATTERN = Pattern.compile("(\\{[a-zA-Z]+\\})");
+    private static final Pattern PARAMETER_PATTERN = Pattern.compile("(\\{[a-zA-Z0-9]+[a-z-_A-Z0-9]*\\})");
```

Comment:
can parameter end with "-" ?

Reply:
In the  openapi specification there is no clear mention, but in the API Manager we could assign such variables as well.